app/views/shared/_language_fields.html.haml

```diff
@@ -1,4 +1,4 @@
 .nested-fields
   %fieldset.inputs
-    = f.input :code, as: :language_select, priority: %i(de en es)
+    = f.input :code, as: :language_select, priority: %i(de en fr es)
```

Comment:
`ApplicationController#set_locale` also include `pt-BR`.  Should not it be part of this list (and maybe both lists could be a single object)?